src/hotspot/share/opto/regmask.cpp

```diff
@@ -67,6 +67,9 @@ int RegMask::num_registers(uint ireg) {
       return SlotsPerVecX;
     case Op_VecD:
       return SlotsPerVecD;
+#if defined(AMD64) || defined(IA32)
+    case Op_RegVMask:
+#endif
```

Comment:
I think, it would be better we have less #ifdef in the shared code. So SlotsPerVMask? :-)